<launch>
    <arg name="image_topic" default="/camera/left/image_raw" />
    
    <arg name = "calibration_file" default = "$(find calibration_publisher)/calibr_file/2020-06-03-09-59-autoware_lidar_camera_calibration.yaml" />

    <arg name="lidar_frame" default="rslidar"/>
    
    <arg name="camera_frame" default="left_frame"/>

    <node pkg="calibration_publisher" type="calibration_publisher" name="calibration_publisher_node">      

        <param name="image_topic" value="$(arg image_topic)"/>
        
        <param name="calibration_file" value="$(arg calibration_file)"/>

        <param name="lidar_frame" type="str" value="$(arg lidar_frame)"/>

        <param name="camera_frame" type="str" value="$(arg camera_frame)"/> 
    </node>
</launch>
